libraries/botbuilder-lg/src/LGFileLexer.g4

```diff
@@ -12,7 +12,7 @@ lexer grammar LGFileLexer;
 
 @lexer::members {
   ignoreWS = true;             // usually we ignore whitespace, but inside template, whitespace is significant
-  expectIfElse = false;        // whether we are expecting IF/ELSEIF/ELSE
+  expectConditions = false;        // whether we are expecting IF/ELSEIF/ELSE
```

Comment:
>// whether we are expecting IF/ELSEIF/ELSE [](start = 35, length = 42)

update comments here

Reply:
let's also rename Conditions to a better name, because we are not expecting conditions itself, we are expecting the keyword before that, so expectControlStatement? expectKeywords?

Reply:
Will change to expect keywords. And the comments will be updated correspondingly.